kitt_bar_app/view.rb

```diff
@@ -1,5 +1,6 @@
 class View
   def append_with(args = {})
+    args[:size] = 11 unless args[:size]
```

Comment:
all text is size 11 now unless we want a different size (ie. for batch titles in the menu)

Reply:
Good Style 🥳